### PR TITLE
PROD-381/add-bonk-claimed-and-cards-revealed

### PR DESCRIPTION
**Pull request by danijel-radomiljac**
Screenshot 2024-10-08 at 15 43 51

@gatorthana I need a icon names for bonk claimed and cards revealed.

### CODE REVIEW

File: app/queries/home.ts [hunk 1]
```diff
@@ -375,3 +381,87 @@ async function queryUserStatistics(userId: string): Promise<UserStatistics> {
...
+
+export async function getUsersTotalClaimedAmount(): Promise<number> {
+  const payload = await authGuard();
+
+  const totalClaimedAmount = await queryUsersTotalClaimedAmount(payload.sub);
+
+  return totalClaimedAmount;
+}
+
+async function queryUsersTotalClaimedAmount(userId: string): Promise<number> {
```
Comment (marvinmarnold) [inline app/queries/home.ts:439]: Please add tests.

**Review by marvinmarnold — CHANGES_REQUESTED** [review]

File: app/queries/home.ts (hunk 1 above)
Comment (danijel-radomiljac) [reply app/queries/home.ts:439]: I will wait for https://github.com/gator-labs/chomp/pull/579 PR to be merged. Because there are some setup test changes.

Comment (yash431garg) [reply app/queries/home.ts:439]: Will try to merge it asap.

**Comment by gatorthana** [comment]
@danijel-radomiljac the BONK icon and the CARD icons were custom ones because I couldn't find a suitable one from Lucide library. @Yihwan for custom icons, would that break things or add too much complications?

**Comment by gatorthana** [comment]
Bonk Icon
card

@danijel-radomiljac here are the SVG for the 2 custom icons, lmk if these are okay!

**Comment by marvinmarnold** [comment]
@danijel-radomiljac conflicts

@neel-ds @yash431garg @AndrejRadomiljac review please

**Pull request merged**